kaguya

blue2wo opened the pull request:
課題 - 家具屋

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+    <div id="global-nav-wrapper">
+        <div id="global-nav">
+            <div id="flex-test">
+                <p><span class="text-break">HTML</span>CSS</p>
```
ここはロゴ画像をXDからとってきてください。

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+            <div id="flex-test">
+                <p><span class="text-break">HTML</span>CSS</p>
+            </div>
+            <img height="30" src="kaguya/images/LogoMakr_4bISVm.png" alt="hamburger menu">
```
PC版にハンバーガーメニューの表示はありません

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+            </div>
+            <img height="30" src="kaguya/images/LogoMakr_4bISVm.png" alt="hamburger menu">
+            <ul id="global-nav-list">
+                <li class="global-nav-item"><a href="#">Home</a></li>
+                <li class="global-nav-item"><a href="#">News</a></li>
+                <li class="global-nav-item"><a href="#">Recruit</a></li>
+                <li class="global-nav-item"><a href="#">Contact</a></li>
```
今回はきちんと遷移する必要があるので#ではない遷移先を指定してください

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+    </div>
+    <div id="local-header">
+        <!-- 画像はCSSの backgroundプロパティで挿入します -->
+        <h1><span>We</span> <span>propose</span> <span>a life.</span></h1>
```
`span`タグを使用されたのはどういう意図がありますか？

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+            <div>
+                <h2>暮らしを提案する。</h2>
+                <p>
+                    <span class="text-break">私たちは、家具を売るのではなく、</span>
+                    <span class="text-break">お客様の理想とする</span>
+                    <span class="text-break">「暮らし」を提案します。</span>
+                    <span class="text-break">仕事で疲れた後に帰りたくなる、</span>
+                    <span class="text-break">今日も１日頑張ろうと思える、</span>
+                    <span class="text-break">そんな空間をお客様と一緒に作り上げていくのが</span>
+                    <span class="text-break">私たちの仕事です。</span>
```
`span`タグである必要がないです。

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+                <input class="contact-item" type="text" placeholder="お名前">
+                <input class="contact-item" type="text" placeholder="メールアドレス">
+                <textarea class="contact-item" id="contact-comment" placeholder="お問い合わせ内容"></textarea>
+                <button class="button-rounded" type="submit" form="contact-form" value="Send">Send</button>
```
`value`に値を入れているので、タグで「Send」を挟む必要はないです。
また、`form`内の送信ボタンなので`button`タグではないです。
あと、「Send」は全て大文字です。

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+                    <p>アームストロングのテーブル入荷</p>
+                </div>
+            </div>
+            <a class="view-more" href="#">view more<span>&gt;</span></a>
```
view moreの遷移先も設定してください。

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+            <div id="global-footer-nav-inner">
+                <p><span class="text-break">HTML</span>CSS</p>
+                <ul>
+                    <li>Home</li>
+                    <li>News</li>
+                    <li>Recruit</li>
+                    <li>Contact</li>
```
ここもリンクにしましょう

piasoratech commented on indexkaguya.html:
```diff
@@ -0,0 +1,127 @@
+                    <li>Contact</li>
+                </ul>
+            </div>
+            <p id="copyright">&copy;HTMLCSS.code</p>
```
コピーライトに適したタグがあるので調べて使用しましょう

blue2wo commented:
課題 - 家具屋(Home HTML) push 2
54a046c

全て修正しました。

choimake commented:
修正ありがとうございました。LGTMです！

blue2wo commented:
課題 - 家具屋(Home News, Recruit, Contact) HTML

ee84611

choimake commented on recruit.html:
```diff
@@ -0,0 +1,110 @@
+        </div>
+    </header>
+    <main>
+        <div class="recruit-wrapper">
+        <div class="recuit">
+                <table class="recruit-info-table">
```
`<div class="recuit">`の開始タグ、終了タグのインデントを修正してください。

choimake commented on news.html:
```diff
@@ -0,0 +1,72 @@
+                        <p>アームストロングのカウチ入荷</p>
+                        <img src="kaguya/images/news02.png" alt="">
+                    </div>                
+                    <div class="news-item">
+                        
+                        <p>アームストロングのカウチ入荷</p>
+                        <img src="kaguya/images/news03.png" alt="">
```
`<div class="news-item">`と`<p>アームストロングのカウチ入荷</p>`の間のスペースは不要ですので削除してください。
同じようなソースがある場合は、そちらも合わせて修正お願いします。

choimake commented on news.html:
```diff
@@ -0,0 +1,72 @@
+            <div class="global-nav">
+                <img class="nav-image-html" src="kaguya/images/logo_header.png" alt="">
+                <ul class="global-nav-list">
+                    <li class="global-nav-item"><a href="homepage.html">Home</a></li>
```
homeへのリンクが正常ではなく、リンクエラーが発生します。
こちらの修正お願いします。
ソースコード内に複数同じ不具合が見受けられますので、そちらも合わせて修正お願いします。

blue2wo commented:
課題 - 家具屋(Home News, Recruit, Contact) HTML - change

全て修正しました。
1ef27d2

piasoratech commented on contact.html:
```diff
@@ -0,0 +1,63 @@
+            <div class="global-nav">
+                <img class="nav-image-html" src="kaguya/images/logo_header.png" alt="">
+                <ul class="global-nav-list">
+                    <li class="global-nav-item"><a href="index.html">Home</a></li>
+                    <li class="global-nav-item"><a href="news.html">News</a></li>
+                    <li class="global-nav-item"><a href="recruit.html">Recruit</a></li>
+                    <li class="global-nav-item"><a class="current-page" href="contact.html">Contact</a></li>
```
ここ全部大文字ですね(他のページも確認して修正をお願いします)

piasoratech commented on contact.html:
```diff
@@ -0,0 +1,63 @@
+                <div class="global-footer-nav-inner">
+                    <img class="nav-footer-image-html" src="Kaguya/images/logo_footer.png" alt="">
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="news.html">News</a></li>
+                        <li><a href="recruit.html">Recruit</a></li>
+                        <li><a href="contact.html">Contact</a></li>
```
ここもすべて大文字です

piasoratech commented on contact.html:
```diff
@@ -0,0 +1,63 @@
+    <header>
+        <div class="global-nav-wrapper">
+            <div class="global-nav">
+                <img class="nav-image-html" src="kaguya/images/logo_header.png" alt="">
```
`alt`タグに適切な値を設定しましょう

piasoratech commented on contact.html:
```diff
@@ -0,0 +1,63 @@
+        <div class="global-footer-nav-wrapper">
+            <div class="global-footer-nav">
+                <div class="global-footer-nav-inner">
+                    <img class="nav-footer-image-html" src="Kaguya/images/logo_footer.png" alt="">
```
`alt`タグに適切な値を設定しましょう(他の`img`タグを使用している部分も同様です。)
以下のサイトを参考にすると良いと思います。
https://www.kanzaki.com/docs/html/htminfo-alt.html

piasoratech commented on news.html:
```diff
@@ -0,0 +1,71 @@
+                        <img src="kaguya/images/AdobeStock_59829158.png" alt="">   
+                    </div>              
+                    <div class="news-item">
+                        <p>アームストロングのカウチ入荷</p>
```
文章が見本と異なります

piasoratech commented on contact.html:
```diff
@@ -0,0 +1,63 @@
+                <h1>Contact</h1>
+            </section>
+        </div>  
+    </header>
```
`header`タグの内容はCONTACTまでかと(他のページも確認して修正をお願いします)

blue2wo commented:
課題 - 家具屋(Home News, Recruit, Contact) HTML - change 3
c16e8fd

> ここ全部大文字ですね(他のページも確認して修正をお願いします)
- 修正しました

> ここもすべて大文字です
- 修正しました

> `alt`タグに適切な値を設定しましょう
- 修正しました

> `alt`タグに適切な値を設定しましょう(他の`img`タグを使用している部分も同様です。)
以下のサイトを参考にすると良いと思います。
- 修正しました

> 文章が見本と異なります
- 修正しました

> headerタグの内容はCONTACTまでかと(他のページも確認して修正をお願いします)
- 修正しました

blue2wo commented:
課題 - 家具屋(Home News, Recruit, Contact) HTML - change 3
c16e8fd

3a2acd9 - added missed `alt` tags

> ここ全部大文字ですね(他のページも確認して修正をお願いします)
- 修正しました

> ここもすべて大文字です
- 修正しました

> `alt`タグに適切な値を設定しましょう
- 修正しました

> `alt`タグに適切な値を設定しましょう(他の`img`タグを使用している部分も同様です。)
以下のサイトを参考にすると良いと思います。
- 修正しました

> 文章が見本と異なります
- 修正しました

> headerタグの内容はCONTACTまでかと(他のページも確認して修正をお願いします)
- 修正しました

choimake commented on index.html:
```diff
@@ -54,15 +54,15 @@ <h2>暮らしを提案する。</h2>
                     <div class="news-item">
                         <p>アームストロングのテーブル入荷</p>
-                        <img src="kaguya/images/news03.png" alt="">
+                        <img src="kaguya/images/news03.png" alt="stocked item">
```
これらの画像はそれぞれ別の商品の画像なので、altも別の値を設定してください。
他にも同じ値を設定している場所があるので、そちらもあわせて修正してください。

blue2wo commented:
課題 - 家具屋(Home News, Recruit, Contact) HTML - change 4

> これらの画像はそれぞれ別の商品の画像なので、altも別の値を設定してください。
他にも同じ値を設定している場所があるので、そちらもあわせて修正してください。
- 修正しました

7eb5480

blue2wo commented:
課題 - 家具屋(Home News, Recruit, Contact) HTML - change 4

68642b8　－ updated `alt` tags

7eb5480
これらの画像はそれぞれ別の商品の画像なので、altも別の値を設定してください。
他にも同じ値を設定している場所があるので、そちらもあわせて修正してください。

修正しました

blue2wo commented:
課題 - 家具屋(Home CSS提出) push 1
7ace50f

yusaku-inamura commented on style.css:
```diff
@@ -0,0 +1,280 @@
+    background: url(kaguya/images/home_top@2x.png);
+    background-position: center center;
+    background-size: cover;
+    height: calc(100vh - 60px);
```
viewportから60pxを引いて高さを指定してしまうと、ブラウザの表示領域が変わるとレイアウトが崩れてしまいます。
自分の環境でもブラウザのサイズを変更して確認し、修正してみてください。

yusaku-inamura commented on style.css:
```diff
@@ -0,0 +1,280 @@
+.global-footer-nav{
+    background-color: #918F89;
+    color: #f3f3f3;
+    height: 30vh;
```
ここも同じですね。
表示領域に対する高さ30%という指定だと表示領域の大きさに応じてフッター のレイアウトがずれてしまいます。

blue2wo commented:
課題 - 家具屋(Home CSS提出) push 2
修正したと思います
99baa7c

choimake commented:
- 「アームストロングのカウチ入荷」「アームストロングのチェア再入荷」「アームストロングのテーブル入荷」には`text-shadow`のstyleが必要です。

XDのデータから`text-shadow`の設定を取得し、コードに反映してください。

<img width="1050" alt="スクリーンショット 2020-04-23 11 36 00" src="https://user-images.githubusercontent.com/52233450/80052895-ad560980-8556-11ea-835a-4adc3a2baccb.png">

- 以下の画像の「お名前」「メールアドレス」「お問い合わせ内容」のフォントがXDで指定されたデータと異なります。
フォントをXDのデータに合わせて修正してください。

<img width="485" alt="スクリーンショット 2020-04-23 11 39 24" src="https://user-images.githubusercontent.com/52233450/80053217-57359600-8557-11ea-8721-677684fa5a4b.png">

blue2wo commented:
3a8f122

> XDのデータからtext-shadowの設定を取得し、コードに反映してください。
- 修正しました。

> 以下の画像の「お名前」「メールアドレス」「お問い合わせ内容」のフォントがXDで指定されたデータと異なります。
フォントをXDのデータに合わせて修正してください。
- 修正しました

choimake commented:
全体的に適応されているフォントが「Notp Sans JP」になっていますが、「Noto Sans」が正しいフォントです。
XDのデータを確認して、全体的にフォントの種類やweightなどを修正してください。

blue2wo commented:
072fa42
それが正しいと思う.
それが正しくない場合、私は理解していない. ごめんなさい。

- added missing CSS


- Changed Font from [Noto Sans JP] to [Noto Sans] 
-フォントを"Noto Sans JP"から"Noto Sans JP"に変更]

choimake commented:
フォントについてです。前回の段階で指摘できておらず申し訳ありません。

「NEWS」「公式カタログ」などの色んなテキストで`letter-spacing` のstyleが適応されていません。XDのデータを確認しstyleの適応をしてください。
他にも該当箇所があるので、合わせて修正お願いします。

<img width="1050" alt="スクリーンショット 2020-04-23 11 36 00" src="https://user-images.githubusercontent.com/52233450/80063318-a20fd780-8570-11ea-81b0-c0b888b740c6.png">

blue2wo commented:
ab8e532
XDのデータを確認しstyleの`チェックした。それが正しいと思います。

choimake commented:
修正ありがとうございました。LGTMです！

blue2wo commented:
家具屋のrecruitとnewsとcontactのCSSを追加しました。
3889557

choimake commented:
footerの各文字の間隔がXDのデザインより開いています。
前回の指摘漏れで申し訳ありません。こちら修正お願いします。

<img width="619" alt="スクリーンショット 2020-04-24 11 52 33" src="https://user-images.githubusercontent.com/52233450/80170635-59fdbd00-8623-11ea-9cd2-4a94a0e295f0.png">

こちら枠線の太さや枠線が切れているなど、XDのデザインと比べて色々違う点がありますので、こちら修正お願いします。

<img width="676" alt="スクリーンショット 2020-04-24 11 52 41" src="https://user-images.githubusercontent.com/52233450/80170689-7d286c80-8623-11ea-8ca5-e6911fdc94bb.png">

blue2wo commented:
修正したと思います。
もしまだ間違っていたら教えて下さい... we'll see :)
b5f7c29

choimake commented:
HOME画面が画像の位置からスクロールできなくなっています。
こちら修正お願いします。

<img width="614" alt="スクリーンショット 2020-04-24 13 41 16" src="https://user-images.githubusercontent.com/52233450/80175815-aea83480-8631-11ea-9861-ccbb656c6c55.png">

blue2wo commented:
0232a94

> HOME画面が画像の位置からスクロールできなくなっています。
> こちら修正お願いします。
修正しました　- added space at the bottom of the page.

choimake commented:
対応ありがとうございました。LGTMです。
